source/guide_ttrss.rst

```diff
@@ -0,0 +1,200 @@
+
+License
+=======
+
+The software is licensed under the GPL. All relevant information can be found in the repository of the project. 
+
+  * https://git.tt-rss.org/git/tt-rss
+
+Prerequisites
+=============
+
+We're using PHP in the latest stable version 7.1:
+
+::
+
+ [isabell@stardust ~]$ uberspace tools version show php
+ Using 'PHP' version: '7.1'
+ [isabell@stardust ~]$
+
+.. include:: includes/my-print-defaults.rst
+
+The URL where you want to run Tiny Tiny RSS needs to be setup:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+Clone the Tiny Tiny RSS repository
+----------------------------------
+
+As Tiny Tiny RSS uses a rolling release model with the git master branch as current stable version, we
+wil use git to download the software. So go to the webroot and clone Tiny Tiny RSS into the subfolder tt-rss
+
+::
+
+ [isabell@stardust ~]$ cd ~/html
+ [isabell@stardust ~]$ git clone https://tt-rss.org/git/tt-rss.git tt-rss
+ Klone nach 'tt-rss' ...
+ remote: Counting objects: 59813, done.
+ remote: Compressing objects: 100% (23687/23687), done.
+ remote: Total 59813 (delta 32579), reused 59789 (delta 32557)
+ Empfange Objekte: 100% (59813/59813), 74.56 MiB | 1.28 MiB/s, Fertig.
+ Löse Unterschiede auf: 100% (32579/32579), Fertig.
+ [isabell@stardust ~]$
+
+Configuration
+=============
+
+Setup your database
+-------------------
+
+It's a good idea to create the tables under a separate database.
+
+::
+
+ [isabell@stardust ~]$ mysql -e "CREATE DATABASE isabell_tt_rss"
+
+Replace <isabell> with your own user. 
+
+Configure database and domain
+-----------------------------
+
+Now you should reach your installation under the URL ``https://<username>.uber.space/tt-rss`` (would be ``https://isabell.uber.space/tt-rss`` in our example).
+Open the URL and set the Database settings to
+
+ * Database Type: MySQL
+ * Username: isabell (your MySQL username)
+ * Password: MySuperSecretPassword (your MySQL password)
+ * Database name: isabell_tt_rss (replace isabell with your own user)
+
+Under other settings the domain https://isabell.uber.space/tt-rss should be shown.
+Now click on `Test Configuration`. If everything worked you should see the messages
+ * Configuration check succeeded.
+ * Database test succeeded.
+
+Click on `Initialize Database`. After that the generated configuration will be shown. Click on `Save configuration` to save it.
+If you need to change it afterwards you will find it under ``~/html/tt-rss/config-php``.
+
+ Configure users
+ ---------------
+
+ Now you can reach Tiny Tiny RSS under https://isabell.uber.space/tt-rss for the first time. Log in with 
+
+  * User: admin
+  * Password: password
+
+You will be greeted by a dialog to change the default password. Do that and optionally add one or more users with restricted rights
+for capturing and reading RSS feeds.
+
+Setup daemon to fetch RSS feeds
+-------------------------------
+Create ``~/etc/services.d/ttrss.ini`` with the following content:
```

Comment:
I think it would be better to give a general example and then specify it with your example like [here](file:///Users/mo/Downloads/lab/build/html/guide_ghost.html#setup-daemon)